core/src/main/resources/error/error-classes.json

```diff
@@ -134,9 +154,41 @@
     "message" : [ "The second argument of '%s' function needs to be an integer." ],
     "sqlState" : "22023"
   },
+  "TABLE_STATS_MUST_SPECIFIED" : {
+    "message" : [ "table stats must be specified." ],
+    "sqlState" : "42000"
+  },
+  "UNABLE_CREATE_DATABASE_AS_FAILED_CREATE_DIRECTORY" : {
```

Comment:
We can probably consolidate `UNABLE_CREATE_DATABASE_AS_FAILED_CREATE_DIRECTORY` and `UNABLE_CREATE_TABLE_AS_FAILED_CREATE_DIRECTORY`, i.e.:

```
"CANNOT_CREATE_OBJECT_DUE_TO_FAILED_DIRECTORY_CREATION": {
  "message": [ "Unable to CREATE %s %s as failed to create its directory %s. " ]
}
```

Where this is called with:
```
new SparkException("CANNOT_CREATE_OBJECT_DUE_TO_FAILED_DIRECTORY_CREATION",
      Array("DATABASE", dbDefinition.name, dbDefinition.locationUri.toString), e)
```